docs/source/release_notes.rst

```diff
@@ -7,12 +7,13 @@ Future Release
     * Enhancements
     * Fixes
     * Changes
+        * Added notebook standardizer to remove executed outputs (:pr:`1153`)
```

Comment:
I'm thinking we should include this in the `Testing Changes` section instead of here since it will really be used as part of the CI checks to make sure the documents are in good order.